contracts/migrations/2_deploy_v0.js

```diff
@@ -24,8 +35,10 @@ async function deploy(options, accounts) {
     { from: accounts[0] }
   );
 
+  let noLossDaoInstance = await NoLossDao_v0.at(noLossDao.address);
```

Comment:
I'm not sure exactly why, but the interface for the `noLossDao` variable didn't contain the `initialize` function. That is why I am getting the instance like this. I think the thing returned by OpenZeppelin is just the proxy.